testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/tableProperties.tests.js

```diff
@@ -210,6 +218,40 @@ module('Table properties forms', {
             assert.strictEqual($tableElement.css('textAlign'), 'right', 'text align is applied');
         });
 
+        test('Passing backgroundColor & borderColor from empty selected cells to colorBox', function(assert) {
+            this.createWidget({ value: tableWithoutContent });
+
+            const $tableElement = this.$element.find('table').eq(0);
+
+            this.quillInstance.setSelection(0, 2);
+            showCellPropertiesForm(this.instance, $tableElement);
+            this.clock.tick();
+
+            const formInstance = this.getFormInstance();
+            const backgroundColorEditor = formInstance.$element().find('.dx-colorbox').eq(1).dxColorBox('instance');
+            const borderColorEditor = formInstance.$element().find('.dx-colorbox').eq(0).dxColorBox('instance');
+
+            assert.strictEqual(backgroundColorEditor.option('value'), 'green', 'background color passed to colorBox');
+            assert.strictEqual(borderColorEditor.option('value'), 'gray', 'border color passed to colorBox');
+        });
+
+        test('Apply backgroundColor & borderColor of first selected call for all cells when color did not modified in colorBox', function(assert) {
+            this.createWidget({ value: tableWithoutContent });
+
+            const $tableElement = this.$element.find('table').eq(0);
+
+            this.quillInstance.setSelection(0, 2);
+            showCellPropertiesForm(this.instance, $tableElement);
+            this.clock.tick();
+
+            this.applyFormChanges();
+
+            assert.strictEqual($tableElement.find('td').eq(0).css('backgroundColor'), 'rgb(0, 128, 0)', 'background color is applied');
+            assert.strictEqual($tableElement.find('td').eq(0).css('borderColor'), 'rgb(128, 128, 128)', 'borderColor color is applied');
+            assert.strictEqual($tableElement.find('td').eq(1).css('backgroundColor'), 'rgb(0, 128, 0)', 'background color is applied');
+            assert.strictEqual($tableElement.find('td').eq(1).css('borderColor'), 'rgb(128, 128, 128)', 'borderColor color is applied');
+        });
+
```

Comment:
don't we want test that first selected cell's color is passed to colorBox during editing?

Reply:
it's checked [here](https://github.com/DevExpress/DevExtreme/pull/23825/files#diff-d6397b99967eae72a6fe372a0d6477ffdb29c874322016ce4e2e3318fd4766f5R221)